lib/rage/router/dsl.rb

```diff
@@ -129,6 +129,54 @@ def match(path, to:, constraints: {}, defaults: nil, via: :all)
       end
     end
 
+    def resources(name, constraints: nil, defaults: nil, only: nil, except: nil)
```

Comment:
Hey @arikarim 

I've worked on the `resources` helper in another PR. Do you want to check it out?

Reply:
Of course,  I will delete the resources part in this one.